### PR TITLE
Create Recipe Card

### PR DESCRIPTION
**Pull request by Cryszzz**
I have made a recipe card as well as an expanded version. (same as Lab 7 of how to use it)
BE CAREFUL: incompatible with JSON file in our repo. 

#46

### CODE REVIEW

**Comment by PatrickBrown1** [comment]
Looks really good so far!

I went through and fixed all of the styling errors so it's passing the tests, however, we aren't completely done. 

Now that we have the recipe schema (in ```./root/models/recipes.json``` ), can you go through and change the util functions (now in ```./root/scripts/RecipeCardUtil.js```) to work with the new recipe schema?

**Review by PatrickBrown1 — CHANGES_REQUESTED** [review]
See above comment

File: root/component/RecipeCard.js [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+    image.setAttribute('alt', titleText);
+
+    // Grab the organization name
+    const organizationText = getOrganization(data);
+    const organization = document.createElement('p');
+    organization.classList.add('organization');
+    organization.innerText = organizationText;
+
+    // Grab the reviews
+    const ratingVal = searchForKey(data, 'ratingValue');
```
Comment (PatrickBrown1) [inline root/component/RecipeCard.js:146]: We don't have a rating for each recipe, so we don't need this part

**Review by PatrickBrown1 — CHANGES_REQUESTED** [review]

**Comment by PatrickBrown1** [comment]
Fixed recipe card component to work with recipe data, fixed styles so that it follows figma, removed unneeded components.

image

**Review by gmihir — APPROVED** [review]

**Review by PatrickBrown1 — APPROVED** [review]

**Pull request merged**